bindings/ios/MEGARequest.mm

```diff
@@ -193,4 +208,14 @@ - (NSInteger)numDetails {
     return self.megaRequest ? self.megaRequest->getNumDetails() : 0;
 }
 
+- (NSArray<MEGAStringList *> *)megaStringTableArray {
+    MegaStringTable *table = self.megaRequest ? self.megaRequest->getMegaStringTable() : nil;
+    NSMutableArray<MEGAStringList *> *array = NSMutableArray.alloc.init;
```

Comment:
@jcmartinac , since we know the size here, please use `arrayWithCapacity:` method.